resources/eventing/charts/event-publisher-proxy/templates/deployment.yaml

```diff
@@ -0,0 +1,71 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "event-publisher-proxy.fullname" . }}
+  labels:
+  {{- include "event-publisher-proxy.labels" . | nindent 4 }}
```

Comment:
As per helm docs, [nindent](https://helm.sh/docs/chart_template_guide/function_list/#nindent) adds a new line to the beginning of the section, which renders as the following:

```yaml
metadata:
  labels:

    key1: value1
    key2: value2
```

Is this what we want? 

Alternatively, we can use `indent` and make sure to remove whitespaces before it, as the following:

```yaml
metadata:
  name: {{ include "event-publisher-proxy.fullname" . }}
  labels:
{{- include "event-publisher-proxy.labels" . | indent 4 }}
```

Reply:
We agreed to use `nindent` inlined with the desired key, as the following:

```yaml
metadata:
  labels: {{- include "event-publisher-proxy.labels" . | nindent 4 }}
```